be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx

```diff
@@ -136,7 +136,7 @@
     <value>A customer with this e-mail address already exists.</value>
   </data>
   <data name="ClientManagers" xml:space="preserve">
-    <value>Client  account managers</value>
+    <value>Client account managers</value>
```

Comment:
**suggestion:** there are probably some more labels to have a look at:

![image](https://user-images.githubusercontent.com/7514974/173429145-70b653de-052c-47d4-bb60-f99c12f4e5c9.png)